main.go

```diff
@@ -51,6 +52,7 @@ func init() {
 	flag.StringVar(&copyOptions, "copy-options", "", "Additional options to pass to COPY (ex. NULL 'NULL')")
 	flag.StringVar(&splitCharacter, "split", ",", "Character to split by")
 	flag.StringVar(&fromFile, "file", "", "File to read from rather than stdin")
+	flag.StringVar(&columns, "columns", "", "Columns to present in CSV")
```

Comment:
Would change the description to:
"Comma-separated columns present in CSV"

Just so people know to use commas between the columns.